api/spec/models/spree/api/webhooks/stock_item_decorator_spec.rb

```diff
@@ -0,0 +1,151 @@
+require 'spec_helper'
+
+describe Spree::StockItem do
+  describe 'sending webhooks' do
+    let(:queue_requests) { instance_double(Spree::Webhooks::Subscribers::QueueRequests) }
```

Comment:
It seems you don't need to define this double here because is defined in the `emit_webhook_event` matcher.